### PR TITLE
Set target to Java 8 (for 1.13) & update Maven plugins [broken]

### PR DESCRIPTION
**Pull request by DoNotSpamPls**
The difference between the last PR and this one is that this one applies the changes to the 1.13 branch, instead of the 2.x one.

I compiled with the new plugins on my PC, and it compiled successfully, so there shouldn't be any issues.

### CODE REVIEW

File: EssentialsGeoIP/pom.xml [hunk 1]
```diff
@@ -18,31 +18,25 @@
...
+            </dependency>
+            <dependency>
+                <groupId>com.maxmind.geoip</groupId>
+                <artifactId>geoip-api</artifactId>
+                <version>1.2.11</version>
+            </dependency>
     </dependencies>
 
     <build>
         <plugins>
```
Comment (caojohnny) [inline EssentialsGeoIP/pom.xml]: Why did you refrain from indenting anything in the `build` tag?

File: nms/1_8_R1Provider/pom.xml [hunk 2]
```diff
@@ -16,7 +16,7 @@
         <repository>
             <id>project.local</id>
             <name>local repo</name>
-            <url>file:${project.basedir}/lib</url>
+            <url>file:${project.basedir}\lib</url>
```
Comment (caojohnny) [inline nms/1_8_R1Provider/pom.xml]: Why was this changed?

**Review by caojohnny — CHANGES_REQUESTED** [review]

File: nms/1_8_R1Provider/pom.xml (hunk 2 above)
Comment (DoNotSpamPls) [reply nms/1_8_R1Provider/pom.xml]: That was changed because my IDE wasn't building the project, it said that it can't find the file. Changing that fixed it.

File: EssentialsGeoIP/pom.xml (hunk 1 above)
Comment (DoNotSpamPls) [reply EssentialsGeoIP/pom.xml]: That wasn't meant to be indented, will fix

File: nms/1_8_R1Provider/pom.xml (hunk 2 above)
Comment (mdcfe) [reply nms/1_8_R1Provider/pom.xml]: This will break the project configuration on any OS aside from Windows :/

Comment (DoNotSpamPls) [reply nms/1_8_R1Provider/pom.xml]: 👌, will revert to the original configuration.

Comment (kashike) [reply nms/1_8_R1Provider/pom.xml]: Is this even needed? I don't see any dependencies in this module that come from this directory.

Comment (DoNotSpamPls) [reply nms/1_8_R1Provider/pom.xml]: This is needed for the craftbukkit 1.8 jar, which is needed for building the project. The directory is obviously not committed due to legal reasons, so you have to do it yourself. Check the readme.md file in nms/1_8_R1Provider

Comment (lucko) [reply nms/1_8_R1Provider/pom.xml]: BuildTools will install the dependency into your local Maven repository, so there's no reason why you need to copy it there.

Comment (DoNotSpamPls) [reply nms/1_8_R1Provider/pom.xml]: I have it in my local Maven repository, yet IntelliJ says it can't find craftbukkit 1.8.
On another note, it seems that 1.13 bukkit is gone from the Spigot repository for some reason.

**Review by caojohnny — APPROVED** [review]

File: nms/1_8_R1Provider/pom.xml (hunk 2 above)
Comment (SupaHam) [reply nms/1_8_R1Provider/pom.xml]: @DoNotSpamPls You have to make sure you have the right dependency of course. 1.8 had many versions.

Comment (DoNotSpamPls) [reply nms/1_8_R1Provider/pom.xml]: Yes, I have the exact version required, but it still only finds it when this repository is added and I put the file where it's requested. /shrug

Comment (SupaHam) [reply nms/1_8_R1Provider/pom.xml]: Then there's a problem with your setup. The project structure works fine for me. So I'd double check that the jar actually exists, maybe your maven isn't setup properly. But I can assure you the project is.

Comment (DoNotSpamPls) [reply nms/1_8_R1Provider/pom.xml]: Okay, this is my problem then. Should I remove that section or let it be?

Comment (mdcfe) [reply nms/1_8_R1Provider/pom.xml]: I'd say leave it unchanged for now.

**Comment by DoNotSpamPls** [comment]
Can the commenters approve or express their other concerns? I think everything has been fixed now.

**Review by SupaHam — APPROVED** [review]

**Comment by mdcfe** [comment]
Why close this?

**Comment by DoNotSpamPls** [comment]
@md678685 It was an accident, I reforked this repo and forgot I still had a PR open. Well, time to recreate this PR i guess (smh)
